CHANGELOG.md

```diff
@@ -7,177 +7,43 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
-- [#10737](https://github.com/MetaMask/metamask-extension/pull/10737): eth-block-tracker@5.0.1
-- [#10663](https://github.com/MetaMask/metamask-extension/pull/10663): security - update SES lockdown
-- [#10718](https://github.com/MetaMask/metamask-extension/pull/10718): build - refactor build system for easier configuration
-- [#10735](https://github.com/MetaMask/metamask-extension/pull/10735): ci - cache deps before patch-package
-- [#10701](https://github.com/MetaMask/metamask-extension/pull/10701): Update @metamask/controllers to v6.2.1
-- [#10714](https://github.com/MetaMask/metamask-extension/pull/10714): fix: remove unused `metamask.rpcUrl` from redux state + fix tests to reflect that
-- [#10636](https://github.com/MetaMask/metamask-extension/pull/10636): Delete setupFetchDebugging.js
-- [#10687](https://github.com/MetaMask/metamask-extension/pull/10687): deps - remove "remotedev-server"
-- [#10682](https://github.com/MetaMask/metamask-extension/pull/10682): Adding default properties to NetworkForm
-- [#10689](https://github.com/MetaMask/metamask-extension/pull/10689): make migration more safe
-- [#10639](https://github.com/MetaMask/metamask-extension/pull/10639): refactor incoming tx controller
-- [#10666](https://github.com/MetaMask/metamask-extension/pull/10666): Fixing incorrectly typed token decimal attribute
-- [#10627](https://github.com/MetaMask/metamask-extension/pull/10627): fix: replace dnode background with JSON-RPC
-- [#10661](https://github.com/MetaMask/metamask-extension/pull/10661): Improve specificity of `test:unit:lax` npm script
-- [#10664](https://github.com/MetaMask/metamask-extension/pull/10664): Fixing migration script generation paths
-- [#10655](https://github.com/MetaMask/metamask-extension/pull/10655): colocate tests in flat structure
-- [#10656](https://github.com/MetaMask/metamask-extension/pull/10656): Allow TextField to receive min and max attributes
-- [#10652](https://github.com/MetaMask/metamask-extension/pull/10652): Sync `master` with `develop`
-- Merge remote-tracking branch 'origin/develop' into sync-master
-- [#10614](https://github.com/MetaMask/metamask-extension/pull/10614): Move swaps constants to the shared constants directory
-- [#10640](https://github.com/MetaMask/metamask-extension/pull/10640): no more node:console
-- [#10634](https://github.com/MetaMask/metamask-extension/pull/10634): Setting balance to 0x0 when the original value is undefined
-- [#10615](https://github.com/MetaMask/metamask-extension/pull/10615): remove transactionCategory in favor of more types
-- [#10524](https://github.com/MetaMask/metamask-extension/pull/10524): Ensure permission log will only store JSON-able data
-- [#10624](https://github.com/MetaMask/metamask-extension/pull/10624): Replace logic for eth swap token in fetchQuotesAndSetQuoteState with getSwapsEthToken call
+- [#10963](https://github.com/MetaMask/metamask-extension/pull/10963): Ensure "Add Contact" button appears when a user already has contacts.
```

Comment:
`[#10963](https://github.com/MetaMask/metamask-extension/pull/10963): Ensure "Add Contact" button appears when a user already has contacts.` should not be included, it was in v9.5.0

Reply:
fixed